tests/test_transforms.py

```diff
@@ -222,7 +246,41 @@ def test_calculate_displacement_from_voltage():
 
     assert isinstance(actual_converted_data, NDArray[(2, Any), np.float32])
 
-    expected_data = [1.1, 0.1, -0.9]
+    # converting test voltage to expected displacements
+    conversion_factor_displacement = 23.25
```

Comment:
to help keep the code DRY ("do not repeat yourself") and have a "single source of truth", typically if there's a constant used in both the codes and the tests, then the tests import it from the main code